Gemfile

```diff
@@ -8,7 +8,7 @@ gem "rails", "~> 6.1.4"
 # Use postgresql as the database for Active Record
 gem "pg", "~> 1.1"
 # Use Puma as the app server
-gem "puma", "~> 5.0"
+gem "puma", "~> 5.5.1"
```

Comment:
[suggestion] this level of specificity will prevent upgrading to `5.6.0` when it is released. If we want to verify that at least `5.5.1` is installed, a better version string would be `gem "puma", "~> 5.5", ">= 5.5.1"`